tsconfig.strictNullChecks.json

```diff
@@ -4,6 +4,7 @@
         "strictNullChecks": true
     },
     "files": [
+        "././src/tests/end-to-end/common/page-controllers/nested-iframe-target-page.ts",
```

Comment:
Extra `./` at the beginning of the new line (it's benign but breaks sorting of the list)